channels/il.m3u

```diff
@@ -1,51 +1,31 @@
 #EXTM3U x-tvg-url="http://epg.streamstv.me/epg/guide-israel.xml.gz"
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Hebrew" tvg-logo="" group-title="",ch 12 קשת
+#EXTINF:-1 tvg-id="Channel 12 Keshet-IS" tvg-name="Channel 12 Keshet-IS" tvg-logo="https://img.mako.co.il/2019/10/03/newlogo12.jpg" group-title="עידן+",Keshet 12 קשת
 http://keshethlslive-lh.akamaihd.net/i/CH12CC_1@732426/index_2200_av-p.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Hebrew" tvg-logo="" group-title="",ch 21 ערוץ הקניות
-http://82.80.192.30/shoppingil_ShoppingIL21TVRepeat/_definst_/smil:ShoppingIL21TV.smil/chunklist_w2043660694_b2128000.m3u8?ttl=1524643697&cdn_token=5a6c4f1270440ae72118c8ae16c58de4%7CUser-Agent=Mozilla/5.0%20(Macintosh;%20Intel%20Mac%20OS%20X%2010_10_2)%20AppleWebKit/600.3.18%20(KHTML,%20like%20Gecko)%20Version/8.0.3%20Safari/600.3.18
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/NmnXn2z.jpg" group-title="News",Channel 2 News
+#EXTINF:-1 tvg-id="Channel 13 Reshet-IS" tvg-name="IL: ISRAEL Channel 13 Reshet" tvg-language="Hebrew" tvg-logo="https://media.reshet.tv/image/upload/v1505910155/logo-final-13-reshet-red_a5lwvy.jpg" group-title="עידן+",Reshet 13 רשת
+https://reshet-live-il.ctedgecdn.net/13tv-desktop/r13_2400.m3u8
+#EXTINF:-1 tvg-id="Kan 11-IS" tvg-name="IL: Kan 11" tvg-language="Hebrew" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/c/c8/Kan11Logo.svg/2880px-Kan11Logo.svg.png" group-title="עידן+",Kan 11 כאן - FHD
+https://kanlivep2event-i.akamaihd.net/hls/live/747610/747610/source1_4k/chunklist.m3u8
+#EXTINF:-1 tvg-id="Kan 23-IS" tvg-name="IL: Kan 23" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/6/6b/KanHinuchit.svg/2880px-KanHinuchit.svg.png" group-title="עידן+",ch 23 החינוכית - FHD
```

Comment:
Please do not use a hyphen in the channel name. The fact is that VLC player shows only what is written after it.
<img width="381" alt="Captura de pantalla 2020-02-01 a las 22 14 34" src="https://user-images.githubusercontent.com/7253922/73597733-4bca6400-4540-11ea-9ea0-c918a573afae.png">

Reply:
Hi
OK.
BTW: can you share m3u syntax definition? couldn't find anywhere.
2/ Can I also add Radio channels to this list, or is it just TV?

Reply:
Here you go: https://github.com/iptv-org/iptv/blob/master/.github/CONTRIBUTING.md